### PR TITLE
Travis CI → GitHub Actions

### PR DESCRIPTION
**Pull request by Gallaecio**
I’ve also defined `PYPI_TOKEN` as a repository secret.

Includes:
- Making Python 3.9 support official
- Porting changes to a function from Scrapy to make it compatible with the latest PyPy version

[See test results](https://github.com/Gallaecio/itemloaders/runs/2419084903).

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py35,py36,py37,py38
+envlist = py,extra-deps
```
Comment (kmike) [inline tox.ini:2]: Heh, I'm still now sure what's the best approach. If you do have all environments installed, and want to test the code against them all (which is a good thing to do), you now need to pass env list to tox - it is unclear which one, and the list might be changing over time. So for this use case not having a list of envs to test against looks less developer friendly.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+        - python-version: 3
+          env:
+            TOXENV: docs
+
```
Comment (rennerocha) [inline .github/workflows/main.yml:14]: **nitpick**: Do we need a line break here? :-) We have others in the next lines too.

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade tox codecov
+      - name: Run tests
+        env: ${{ matrix.env }}
+        run: tox
+      - name: Publish coverage data
+        uses: codecov/codecov-action@v1
+  publish:
```
Comment (rennerocha) [inline .github/workflows/main.yml:55]: **suggestion**: What do you think about creating a separated workflow for publishing (like the one we have on [scrapy](https://github.com/scrapy/scrapy/blob/master/.github/workflows/publish.yml) or [spidermon](https://github.com/scrapinghub/spidermon/blob/master/.github/workflows/publish.yml)?

**Review by rennerocha — COMMENTED** [review]
**suggestion**: Don't forget to update README.md replacing Travis badge.

**Review by wRAR — APPROVED** [review]

**Pull request merged**